File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.cpp
class Solution {
public:
    vector<int> getSneakyNumbers(vector<int>& nums) {
        unordered_map<int,int> ump;
        for(int i=0;i<nums.size();i++){
            ump[nums[i]]++;
        }
        vector<int> ans;
        for(auto x:ump){
            if(x.second==2){
                ans.push_back(x.first);
            }
        }
        return ans;
    }
};

File: 3289-the-two-sneaky-numbers-of-digitville/3289-the-two-sneaky-numbers-of-digitville.py
class Solution:
    def getSneakyNumbers(self, nums: List[int]) -> List[int]:
        ump = {}
        ans = []
        for i in nums:
            ump[i] = ump.get(i,0)+1
            if ump[i]==2:
                ans.append(i)
        return ans